profiles/tests/test_gql_create_my_profile_mutation.py

```diff
@@ -68,7 +66,7 @@ def test_normal_user_can_create_profile(rf, user_gql_client, email_data, profile
     assert dict(executed["data"]) == expected_data
 
 
-def test_normal_user_cannot_create_profile_with_no_primary_email(
+def test_normal_user_can_create_profile_with_no_primary_email(
```

Comment:
This test is now identical to the previous one, except that in the first one the email is primary, and in this one it's not. These tests could be merged into one parameterized test function.